WordPress/Classes/Utility/WPAssetExporter.m

```diff
@@ -40,7 +41,16 @@ - (void)exportAsset:(ALAsset *)asset
    stripGeoLocation:(BOOL)stripGeoLocation
   completionHandler:(void (^)(BOOL success, CGSize resultingSize, NSData *thumbnailData, NSError *error)) handler
 {
-
+    if (!asset.defaultRepresentation) {
+        if (handler) {
+            NSDictionary * userInfo = @{NSLocalizedDescriptionKey:NSLocalizedString(@"The media you are trying to use is not available locally at the moment. If it belongs to a photo stream please try again later.", @"Message that explains to a user that the current asset they selected is not available on the device. This normally happens when user selects a media that belogns to a photostream that needs to be downloaded locally first.")};
```

Comment:
since the concepts of "local" and remote aren't necessarily obvious, let's shoot for a blocker that might help them achieve the result they way:

@"Images owned by someone else in Photo Stream cannot be added to your WordPress. Try saving to your Camera Roll before uploading."

> OK